src/resolver.ts

```diff
@@ -1,58 +1,86 @@
 import { User } from './entity/User';
 import { getRepository } from 'typeorm';
 import { UserInputError } from 'apollo-server';
-import { hash } from 'bcrypt';
+import { compare, hash } from 'bcrypt';
 import { CustomError } from './errors';
-import { formatError } from './errors';
+import jwt = require('jsonwebtoken');
 
 export const resolvers = {
+  Login: {
+    user: (parents, args) => {
+      return parents;
+    },
+    token: () => {
+      const token = jwt.sign('verifyied', 'supersecret');
+      return token;
+    },
+  },
   Query: {
     hello: (): string => {
       return 'hello world';
     },
   },
   Mutation: {
-    createUser: async (_: string, { name, email, password, birthDate }) => {
-      const repository = getRepository(User);
+    createUser: async (_: string, { name, email, password, birthDate, token }) => {
+      const payload = jwt.verify(token, 'supersecret');
+      if (payload == 'verifyied') {
```

Comment:
```suggestion
      if (payload === 'verifyied') {
```

É bom evitarmos `==` porque ele compara apenas o valor lógico, não compara tipos.